explorer/src/components/SearchBar.tsx

```diff
@@ -44,12 +74,7 @@ export function SearchBar() {
           <Select
             autoFocus
             ref={(ref) => (selectRef.current = ref)}
-            options={buildOptions(
-              search,
-              cluster,
-              tokenRegistry,
-              clusterInfo?.epochInfo.epoch
-            )}
+            options={currentBuildOptions}
             noOptionsMessage={() => "No Results"}
```

Comment:
When a user searches for a domain, can you change this to "Looking up domain..." until we know if a domain exists or not?